app/styles/includes/_mixins.scss

```diff
@@ -62,15 +62,15 @@
   margin: 0;
   @include rem('padding', $padding);
   background: none;
-  @include border-radius(0);
+  border-radius: 0;
 }
 
 // Button mixin
 @mixin button($text-color, $bg-color) {
   font: 12px bold sans-serif;
   border: none;
   line-height: 1;
-  @include box-shadow(none);
+  box-shadow: none;
   @include button-color($text-color, $bg-color);
```

Comment:
Why is it that @include font-size, button-color, and media-query were not removed?

Reply:
Those mix-ins were not supplied by Compass

@mattnolting added and maintains those himself:
https://github.com/RedHatInsights/insights-frontend/blob/master/app/styles/includes/_mixins.scss#L137